ADD: adding microsite play

nicovanniekerk opened the pull request:
@renevatium please give this a quick view.

nicovanniekerk committed:
ADD: adding microsite play

renevatium reviewed (commented):
Looks good. Are you using a conditional in the assets role? Otherwise maybe you want to include when: assets==true or whatever the syntax is for the assets role call.

renevatium reviewed (commented):
Also, it is best practice to leave a new line at the end of most files (hence the little red icon)

nicovanniekerk commented:
@renevatium yes the asset playboook is locked to the asset variable in its conditionals.

nicovanniekerk committed:
ADD: microsite execution from update script

nicovanniekerk committed:
ADD: microsite execution from update script

nicovanniekerk committed:
ADD: microsite execution from update script

nicovanniekerk commented:
@renevatium i have added microsites to the update script, with a conditional to ensure you pass the playbook the key it needs to authenticate with github, and also moved the play to the edx-east directory so that it runs with the same context as the other plays called from update. and i added the new line. Please have a gander :)
